jerry-core/ecma/builtin-objects/ecma-builtins.c

```diff
@@ -270,6 +270,39 @@ ecma_finalize_builtins (void)
   }
 } /* ecma_finalize_builtins */
 
+/**
+ * Construct a Function object for specified built-in routine
+ *
+ * See also: ECMA-262 v5, 15
+ *
+ * @return pointer to constructed Function object
+ */
+static ecma_object_t *
+ecma_builtin_make_function_object_for_routine (ecma_builtin_id_t builtin_id, /**< identifier of built-in object
+                                                                                  that initially contains property
```

Comment:
I know it is not new code, but please fix the style of the comments. Write `*` on every line